app/web/features/auth/jail/PasswordSection.tsx

```diff
@@ -0,0 +1,116 @@
+import { TextField, Typography, useMediaQuery } from "@material-ui/core";
```

Comment:
There's a `TextField` component in our `components` folder. If you use that then won't need the `<br />`s to force the input onto a new line.